### DISCUSSION TITLE
请问使用EasyX能够完成项目的需求吗

### DISCUSSION THREAD

**Discussion by ghost** [liusy58/NJU_AdvancedProgramming_SP22#180] [Q&A]
在上次实验课之前就已经用EasyX基本实现了stage1的内容，听了上次实验课之后本来打算用Qt重构一下代码，但是时间好像有点不够，所以想问一下使用EasyX是否能够完成项目的需求，或者说实现后面的功能会不会很困难？

**Comment by liusy58** [comment]
其实EasyX应该可以解决所有的问题，但是考虑到他比较古老，可能需要你自己再写的代码会比较多，但是我认为是完全能够完成项目的要求的。
